Neos.ContentRepository/Classes/Domain/Model/Node.php

```diff
@@ -922,6 +922,7 @@ public function getProperty($propertyName, bool $returnNodesAsIdentifiers = fals
         }
 
         if (
+            /** @phpstan-ignore-next-line i will not touch this code */
```

Comment:
The old Node::getProperty does uuuuuglly things :D And i dont want to appear in its git blame :D